api.json

```diff
@@ -2221,7 +2202,17 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/BadRequestError"
+                  "$ref": "#/components/schemas/GenericError"
+                }
+              }
+            }
+          },
+          "429": {
+            "description": "Rate Limit Exceeded",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericError"
```

Comment:
All these new 429 errors have a schemas/GenericError. Is there a more specific type? If not could we make one so it is more clear to the developer?